### PR TITLE
Add ability to disable reCAPTCHA

### PR DESCRIPTION
**Pull request by bretep**
*[GitHub keywords to close any associated issues](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)*

## Motivation

The enterprise version of reCAPTCHA wasn't working, and we don't need reCAPTCHA. Giving the operator the decision to enable/disable the feature is nice.

## Changelog

Docs update in https://github.com/blockscout/docs/pull/152.

### Enhancements
Ability to disable reCAPTCHA

## Checklist for your Pull Request (PR)

  - [x] I added an entry to `CHANGELOG.md` with this PR
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from silently reappearing again.
  - [ ] I checked whether I should update the docs and did so by submitting a PR to https://github.com/blockscout/docs
  - [x] If I added/changed/removed ENV var, I submitted a PR to https://github.com/blockscout/docs to update the list of env vars at https://github.com/blockscout/docs/blob/master/for-developers/information-and-settings/env-variables.md and I updated the version to `master` in the Version column. Changes will be reflected in this table: https://docs.blockscout.com/for-developers/information-and-settings/env-variables. 
  - [ ] If I add new indices into DB, I checked, that they are not redundant with PGHero or other tools

### CODE REVIEW

**Comment by bretep** [comment]
This commit is currently running on PulseChain Testnet v4

You can test it out here:
https://scan.v4.testnet.pulsechain.com/csv-export?address=0xCe2400835B94d41D5b43Fb5fe729581Df84cF586&type=transactions

File: apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex [hunk 1]
```diff
@@ -180,6 +180,42 @@ defmodule BlockScoutWeb.AddressTransactionController do
...
+      :error ->
+        unprocessable_entity(conn)
+
+      {:error, :not_found} ->
+        not_found(conn)
+
+      {:recaptcha, false} ->
+        not_found(conn)
+    end
+  end
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex:217]: You need to place this clause below clause at https://github.com/blockscout/blockscout/blob/d176aebf821d694dab8fa4e094b7a2e672888454/apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex#L219 as this one will always match so we will never check recaptcha_response

File: apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex [hunk 2]
```diff
@@ -232,6 +268,22 @@ defmodule BlockScoutWeb.AddressTransactionController do
...
+    items_csv(
+      conn,
+      %{
+        "address_id" => address_hash_string,
+        "from_period" => from_period,
+        "to_period" => to_period
+      },
+      AddressTransactionCsvExporter
+    )
+  end
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex:285]: Here I think we can replace two clauses with this one actually
```suggestion
  def transactions_csv(conn, params) do
    items_csv(conn, params, AddressTransactionCsvExporter)
  end
```

**Review by sl1depengwyn — CHANGES_REQUESTED** [review]

File: apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex [hunk 3]
```diff
@@ -180,6 +180,42 @@ defmodule BlockScoutWeb.AddressTransactionController do
...
+      end)
+    else
+      :error ->
+        unprocessable_entity(conn)
+
+      {:error, :not_found} ->
+        not_found(conn)
+
+      {:recaptcha, false} ->
+        not_found(conn)
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex:215]: Also, we need to change it like this, for instance, to match https://github.com/blockscout/blockscout/blob/d176aebf821d694dab8fa4e094b7a2e672888454/apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex#L195
```suggestion
      false ->
        not_found(conn)
```

File: apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex [hunk 4]
```diff
@@ -232,6 +268,10 @@ defmodule BlockScoutWeb.AddressTransactionController do
     )
   end
 
+  def transactions_csv(conn, params) do
+    items_csv(conn, params, AddressTransactionCsvExporter)
+  end
+
```
Comment (sl1depengwyn) [inline apps/block_scout_web/lib/block_scout_web/controllers/address_transaction_controller.ex:274]: The clause below now is not used, we need to remove it
Everything else looks good to me

**Review by sl1depengwyn — APPROVED** [review]

**Review by vbaranov — APPROVED** [review]

**Pull request merged**